<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowPreferences</class>
 <widget class="QWidget" name="WindowPreferences">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>404</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setDefaultButton">
       <property name="text">
        <string>Set default values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="prefsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabModel">
      <attribute name="title">
       <string>Modeling</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="deltaTimeLabel">
           <property name="text">
            <string>Delta time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="deltaTimeValue">
           <property name="decimals">
            <number>10</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="startPointsCoordsLabel">
           <property name="text">
            <string>Start point position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xCoordValue">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-10000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yCoordValue">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zCoordValue">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="trajNumberLabel">
           <property name="text">
            <string>Start points number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="trajNumberValue">
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="pointsNumberLabel">
           <property name="text">
            <string>Trajectory size (in points)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="pointsNumberValue">
           <property name="maximum">
            <number>200000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="startPointsDeltaLabel">
           <property name="text">
            <string>Distance between start points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="startPointsDeltaValue">
           <property name="decimals">
            <number>5</number>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCamera">
      <attribute name="title">
       <string>Camera</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="moveSpeedLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Movement speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="speedMoveSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="sensitivityLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Sensitivity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sensitivitySlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabView">
      <attribute name="title">
       <string>Visualization</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="tailPointsNumberLabel">
           <property name="text">
            <string>Visible tail size (in points)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tailPointsNumberValue">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="timerSpeedLabel">
           <property name="text">
            <string>Time speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timerSpeedValue">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="interpolationLabel">
           <property name="text">
            <string>Interpolation level (in %)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="interpolationDegree">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="arcadeModeCheckBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Arcade mode</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="colorsLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="colorsLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Colors:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="colorsHolderLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="addColorButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="removeColorButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVideo">
      <attribute name="title">
       <string>Recording</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="videoHeightLabel">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="videoHeightValue">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>2160</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="videoWidthLabel">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="videoWidthValue">
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>4096</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>prefsTabWidget</tabstop>
  <tabstop>trajNumberValue</tabstop>
  <tabstop>pointsNumberValue</tabstop>
  <tabstop>xCoordValue</tabstop>
  <tabstop>yCoordValue</tabstop>
  <tabstop>zCoordValue</tabstop>
  <tabstop>sensitivitySlider</tabstop>
  <tabstop>speedMoveSlider</tabstop>
  <tabstop>tailPointsNumberValue</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>setDefaultButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowPreferences</receiver>
   <slot>slot_cancel_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowPreferences</receiver>
   <slot>slot_apply_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addColorButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowPreferences</receiver>
   <slot>addNewColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeColorButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowPreferences</receiver>
   <slot>removeLastColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_cancel_button()</slot>
  <slot>slot_set_default_button()</slot>
  <slot>slot_apply_button()</slot>
  <slot>addNewColor()</slot>
  <slot>removeLastColor()</slot>
 </slots>
</ui>
